string state_input 
string robot_id
---
string state_output
int32 id_client
string[] order
bool success



